src/environments/environment.ts.wes

```diff
@@ -0,0 +1,51 @@
+// The file contents for the current environment will overwrite these during build.
```

Comment:
@jazzmind is this intended to be committed to git or should it be gitignored?

Reply:
should be git ignored... oops